controllers/Comments.controller.js

```diff
@@ -0,0 +1,10 @@
+const {fetchComments} = require("../models/Comments.controller")
+
+exports.getComments = ((req, res, next)=>{
+    fetchComments(req.params.article_id).then((comments)=>{
+        if(comments.code === '42703'){next(comments)}
+        else if(comments.length === 0){res.status(404).send({msg:"Article not found"})}
```

Comment:
Our error handling should be dealt with by our error handling middleware functions. We shouldn't have any logic performed by the controller.